ted_sws/core/adapters/cmd_runner.py

```diff
@@ -112,11 +122,36 @@ def on_end(self):
             now=str(self.end_time),
             time=self.end_time - self.begin_time
         ))
+        self.exit(self.exit_code)
+
+    @classmethod
+    def exit(cls, exit_code=DEFAULT_EXIT_CODE):
+        if exit_code > EXIT_CODE_OK:
+            sys.exit(exit_code)
 
 
 class CmdRunnerForMappingSuite(CmdRunner):
     repository_path: Path
+    mapping_suite_id: str
+    notice_id: List[str]
 
     def is_mapping_suite(self, suite_id):
         suite_path = self.repository_path / Path(suite_id)
         return os.path.isdir(suite_path) and any(f == MS_METADATA_FILE_NAME for f in os.listdir(suite_path))
+
+    @classmethod
+    def _init_list_input_opts(cls, input_val) -> List:
```

Comment:
Why this is class method? Can be this function a simple function ?
Add documentation for this class method.

Reply:
this is a class method because I don't see any usage of it outside of it

Reply:
added description